compiler/resolution/implementForallIntents.cpp

```diff
@@ -1723,6 +1724,105 @@ void implementForallIntents2wrapper(CallExpr* call, CallExpr* eflopiHelper)
 // for the (new) ForallStmt-based representation.
 //
 
+// Forall Intent information about each outer variable.
+class FIinfo {
+public:
+  Symbol* ovar;       // outer variable
+  Symbol* svar;       // shadow variable
+  Symbol* reduceGVar; // reduction class, non-NULL only for reduce intents
+  Symbol* reduceMark; // transitional measure; TODO: remove
+  TFITag  intent;     // the forall intent for this variable
```

Comment:
Thinking about it some more... I'd be OK calling it "TaskIntentTag" if you prefer. The downside is that at the moment it has nothing to do with task intents, although in the future it may.